subsys/ipc/ipc_service/backends/Kconfig.icmsg_with_buf

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig IPC_SERVICE_BACKEND_ICMSG_WITH_BUF
+	bool "ICMSG backend with dynamically allocated buffers"
+	default y
+	depends on MBOX
+	depends on DT_HAS_ZEPHYR_IPC_ICMSG_WITH_BUF_ENABLED
```

Comment:
I disagree. Those dependencies are not needed, since the I am using define from the SPSC that is always available regardless of those configs.

I don't think that DT binding is better. If I do it that way, the user have to take care of the alignment. Why adding more unnecessary work for the user, more responsibilities, since it can be automatically done by inheriting value from SPSC. This is always the same memory, so alignment is the same.

Reply:
I will remove IPC_SERVICE_BACKEND_ICMSG_WITH_BUF_CACHE_ALIGN (always enabled). This module will always inherit SPSC.